src/main/java/de/cas/mse/exercise/SumDigPower.java

```diff
@@ -38,4 +24,34 @@
 
 	}
 
+    private long doHappyNumberCalculation(long happyNumberCandidate) {
```

Comment:
method name is interesting...but we dont know a better one

Reply:
Can we find a method name which describes what kind of calculation happens inside?